### PR TITLE
Enable upstream jaeger storage integration tests in e2e test

### PR DESCRIPTION
**Pull request by arajkumar**
## Description

This commit reuses fixtures from upstream jaeger[1] which is exposed as golang pkg.

Though we use non release version `v1.38.2-0.20221006002917-5bf8a28fe06d`, it doesn't have any other changes except [1] which is safe.

This PR also fixes `TestJaegerStorageIntegration/*/GetOperations` test which was failing due to a typo.

Note: Currently we exclude `FindTraces/Tags_+_Operation_name` test which is still failing after merging both  https://github.com/timescale/promscale/pull/1681 and https://github.com/timescale/promscale/pull/1678. [New bug](https://github.com/timescale/promscale/issues/1697) has been created to address the same.

[1] https://github.com/jaegertracing/jaeger/pull/3944

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [x] CHANGELOG entry for user-facing changes
- [ ] Updated the relevant documentation

### CODE REVIEW

File: pkg/jaeger/store/trace_scan.go [hunk 1]
```diff
@@ -115,7 +121,7 @@ func populateSpan(
 	if err != nil {
 		return fmt.Errorf("making resource tags: %w", err)
 	}
-	pcommon.NewMapFromRaw(attr).CopyTo(resourceSpan.Resource().Attributes())
+	newMapFromRaw(attr).CopyTo(resourceSpan.Resource().Attributes())
```
[the remark this answers is not in the document]
Comment (arajkumar) [reply pkg/jaeger/store/trace_scan.go:124]: @niksajakovljevic yes, `go.opentelemetry.io/collector/pdata` has been updated to v0.61.0 by jaeger 1.38.1.

**Review by niksajakovljevic — APPROVED** [review]

**Comment by arajkumar** [comment]
Similar change in jaeger-clickhouse has been merged.
https://github.com/jaegertracing/jaeger-clickhouse/pull/119

File: pkg/tests/end_to_end_tests/jaeger_store_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+					SpanWriter: spanCopyingWriter{writer},
+					CleanUp: func() error {
+						// Jaeger integration test suite runs each test in an isolated environment.
+						// CleanUp ensures that db starts with clean state for every test run by truncating tables which stores span specific information.
+						for _, table := range []string{
+							"span",
+							"event",
+							"link",
+						} {
+							// ignore any error
```
Comment (cevian) [inline pkg/tests/end_to_end_tests/jaeger_store_integration_test.go:88]: why are we ignoring these errors, seem future tests will get messed up if there is an error

**Review by cevian — APPROVED** [review]
Looks great. One minor optional comment. So much better than the copy-the-code approach :)

File: pkg/tests/end_to_end_tests/jaeger_store_integration_test.go (hunk 2 above)
Comment (arajkumar) [reply pkg/tests/end_to_end_tests/jaeger_store_integration_test.go:88]: Will fixed it. Thanks.

**Review by cevian — APPROVED** [review]

**Pull request merged**